src/System.Collections/tests/Generic/SortedSet/SortedSet.Generic.Tests.cs

```diff
@@ -60,22 +60,37 @@ public void SortedSet_Generic_Constructor_IEnumerable_Null()
         public void SortedSet_Generic_Constructor_IEnumerable_IComparer(EnumerableType enumerableType, int setLength, int enumerableLength, int numberOfMatchingElements, int numberOfDuplicateElements)
         {
             IEnumerable<T> enumerable = CreateEnumerable(enumerableType, null, enumerableLength, 0, 0);
+#if netcoreapp
             SortedSet<T> set = new SortedSet<T>(enumerable, GetIComparer());
             Assert.True(set.SetEquals(enumerable));
+#else
+            // There is a bug in desktop that when the comparer is null it is overriding it to Default, but in one of the usages is not using the global comparer but the one passed in to the constructor.
+            // This is being done to not loose coverage and keep track of this bug.
```

Comment:
Is there a (closed) bug in CoreFX with netfx-port-consider we could link from here?